rs/tests/src/rosetta_tests/tests/neuron_staking.rs

```diff
@@ -1,787 +0,0 @@
-use crate::rosetta_tests::{
```

Comment:
Why are we deleting this entire test suite?

Reply:
That is what the epic about [refactoring system tests](https://dfinity.atlassian.net/browse/FI-1446?atlOrigin=eyJpIjoiNjQzODI3YjhiM2E3NDg5OGEyM2NhZTU1YzU0MjI0N2EiLCJwIjoiaiJ9) is about. We want to migrate them to pocket ic tests as the once that are currently in place are outdated and should be removed.